### PR TITLE
change support for LessEqual to not supported

### PR DESCRIPTION
**Pull request by xu-yuan1**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *CVS-67922*

### CODE REVIEW

**Comment by moslex** [comment]
@ilya-lavrenov , @echuraev , could you please take a look?

**Comment by echuraev** [comment]
@ilyachur could you please take a look? :)

File: docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ The following layers are supported by the plugins:
 | Eltwise-Greater                | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported       |
 | Eltwise-GreaterEqual           | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported       |
 | Eltwise-Less                   | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported\*     |
-| Eltwise-LessEqual              | Supported     |Supported\*\*\*| Supported     | Not Supported | Supported\*     |
+| Eltwise-LessEqual              | Supported     |Supported\*\*\*| Not Supported | Not Supported | Supported\*     |
```
Comment (ilya-lavrenov) [inline docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md:166]: let's drop the whole column for VPU

**Comment by kblaszczak-intel** [comment]
PRs closed due to VPU removal in 23.0. A new PR removes the column: https://github.com/openvinotoolkit/openvino/pull/15270

This PR and additional two regarding VPU will be ported to 22.x versions
https://github.com/openvinotoolkit/openvino/pull/13995
https://github.com/openvinotoolkit/openvino/pull/13997

**Pull request closed without merging**